src/node/internal/internal_fs_callback.ts

```diff
@@ -1247,6 +1250,21 @@ export function createWriteStream(): void {
   throw new Error('Not implemented');
 }
 
+export function glob(
+  _pattern: string | readonly string[],
+  _options:
+    | GlobOptions
+    | GlobOptionsWithFileTypes
+    | GlobOptionsWithoutFileTypes,
+  _callback: ErrorOnlyCallback
+): void {
+  // We do not yet implement the globSync function. In Node.js, this
+  // function depends heavily on the third party minimatch library
+  // which is not yet available in the workers runtime. This will be
+  // explored for implementation separately in the future.
+  throw new ERR_UNSUPPORTED_OPERATION();
```

Comment:
Why not throw `ERR_METHOD_NOT_IMPLEMENTED`?

Reply:
Because I wanted to differentiate between a method that has not been implemented yet vs. one that we are explicitly not supporting yet.